src/components/SocialLinks/SocialLinks.css

```diff
@@ -2,8 +2,17 @@
   margin: 0px 10px;
 }
 
+.social-wrapper {
+  align-self: center;
+  margin-top: 30px;
+}
+
 @media all and (max-width: 576px) {
   .social-link {
     margin: 0 5px;
   }
+
+  .social-wrapper {
+    align-self: center;
```

Comment:
Is it me or this is duplicated? Maybe I am missing something! 🤔

Reply:
It is :)